packages/subsecond/subsecond/src/lib.rs

```diff
@@ -284,6 +284,10 @@ pub struct HotFnPanic {
     _backtrace: backtrace::Backtrace,
 }
 
+/// A pointer to a hot patched function
+#[derive(PartialEq, Eq, Hash, Clone, Copy, Debug)]
+pub struct HotFnPtr(pub u64);
```

Comment:
```suggestion
#[derive(PartialEq, Eq, Hash, Clone, Copy, Debug)]
#[non_exhaustive]
pub struct HotFnPtr(pub u64);
```
I assume this is not meant to be created by consumers

Reply:
this codebase only uses `non_exhaustive` for enums, I would prefer to stick to its coding style